example/ts-types/types.ts

```diff
@@ -0,0 +1,29 @@
+
+// ts-disable
+// prettier-disable 
+export namespace example {
+export type HelloRequest = { 
+	Key: string 
+}
+
+export type HelloResponse = { 
+	Session: string 
+}
+
+export type EchoRequest = { 
+	Session: string  
+	text: string 
+}
+
+//EchoResponse
+export type EchoResponse = { 
+	text: string // 	Text string `json:"text"`
```

Comment:
Does this comment add anything? We already see `json` field ("text") and that it is a string.